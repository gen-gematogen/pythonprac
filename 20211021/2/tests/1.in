:function x x^2+1
:another_function w f"another_function_{w}"
function 1
another_function "foo()"
quit end