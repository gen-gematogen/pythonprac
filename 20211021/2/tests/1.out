2
another_function_foo()
3
